src/back/Update/TopicsDetails.php

```diff
@@ -65,6 +66,10 @@ private function fillDetails(int $beforeUpdate, int $perRun): void
 
             $topics = $this->topics->getUnnamedTopics($perRun);
             if (count($topics)) {
+                $this->logger->debug(
+                    'Запрашиваем сведения о раздачах ({count} шт.) через API (запрос {run}/{runs})...',
+                    ['count' => count($topics), 'run' => $run, 'runs' => $runs]
+                );
```

Comment:
What is the point of this amendment if the number of topics is displayed literally one line lower?

```
07.03.2025 08:34:43 DEBUG: Обновление раздач [19500/57279]
07.03.2025 08:34:51 DEBUG: Обновление раздач [21000/57279]
07.03.2025 08:34:59 DEBUG: Обновление раздач [22500/57279]
07.03.2025 08:35:06 DEBUG: Обновление раздач [24000/57279]
```

Reply:
I agree, this is not needed, thank you!